src/main/java/sirius/biz/storage/layer2/jdbc/SQLVariant.java

```diff
@@ -109,7 +109,7 @@ public class SQLVariant extends SQLEntity implements BlobVariant {
 
     @AfterDelete
     protected void onDelete() {
-        SQLBlob sqlBlob = sourceBlob.fetchValue();
+        SQLBlob sqlBlob = sourceBlob.fetchCachedValue();
```

Comment:
**Question:** Doesn't use of the ref-cache here potentially lead to trying to delete a blob that may be already deleted otherwise?

Reply:
True, will review the places.